ui/helpers/utils/institutional/find-by-custodian-name.ts

```diff
@@ -13,6 +13,7 @@ type Custodian = {
   version: number;
 };
 
+// TODO (Bernardo) - There can be multiple custodian with the same name, envName should be used instead
```

Comment:
I haven't changed anything from this function as it's out of scope for this ticket, but I think it would be best to filter by envName/custodianName.

It's being used in two places and both of them have access to the envName.